media-source-respec.html

```diff
@@ -364,42 +367,47 @@
 
-      <dl title="enum ReadyState" class="idl">
-        <dt>closed</dt>
-        <dd>
+      <div><pre class="idl">enum ReadyState {
+    "closed",
+    "open",
+    "ended"
+};</pre><table class="simple" data-dfn-for="ReadyState" data-link-for="ReadyState"><tbody><tr><th colspan="2">Enumeration description</th></tr><tr><td><dfn><code id="idl-def-ReadyState.closed">closed</code></dfn></td><td>
           Indicates the source is not currently attached to a media element.
-        </dd>
-        <dt>open</dt>
-        <dd>
+        </td></tr><tr><td><dfn><code id="idl-def-ReadyState.open">open</code></dfn></td><td>
           The source has been opened by a media element and is ready for data to be appended to the <a>SourceBuffer</a> objects in <a def-id="sourceBuffers"></a>.
-        </dd>
-        <dt>ended</dt>
-        <dd>
+        </td></tr><tr><td><dfn><code id="idl-def-ReadyState.ended">ended</code></dfn></td><td>
           The source is still attached to a media element, but <a def-id="endOfStream"></a> has been called.
-        </dd>
-      </dl>
+        </td></tr></tbody></table></div>
 
-      <dl title="enum EndOfStreamError" class="idl">
-        <dt>network</dt>
-        <dd>
+      <div><pre class="idl">enum EndOfStreamError {
+    "network",
+    "decode"
+};</pre><table class="simple" data-dfn-for="EndOfStreamError" data-link-for="EndOfStreamError"><tbody><tr><th colspan="2">Enumeration description</th></tr><tr><td><dfn><code id="idl-def-EndOfStreamError.network">network</code></dfn></td><td>
           <p>Terminates playback and signals that a network error has occured.</p>
           <p class="note">JavaScript applications SHOULD use this status code to terminate playback with a network error. For example, if a network error occurs while fetching media data.</p>
-        </dd>
-        <dt>decode</dt>
-        <dd>
+        </td></tr><tr><td><dfn><code id="idl-def-EndOfStreamError.decode">decode</code></dfn></td><td>
           <p>Terminates playback and signals that a decoding error has occured.</p>
           <p class="note">JavaScript applications SHOULD use this status code to terminate playback with a decode error. For example, if a parsing error occurs while processing out-of-band media data.</p>
-        </dd>
-      </dl>
-
-      <dl title="[Constructor] interface MediaSource : EventTarget" class='idl'>
-        <dt>readonly attribute SourceBufferList sourceBuffers</dt>
-        <dd>
+        </td></tr></tbody></table></div>
+
+      <div><pre class="idl">[Constructor]
+interface MediaSource : EventTarget {
+    readonly        attribute SourceBufferList    sourceBuffers;
+    readonly        attribute SourceBufferList    activeSourceBuffers;
+    readonly        attribute ReadyState          readyState;
```

Comment:
Does the respec bug also encompass the enum link I just noted for enum ReadyState?